Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -98,6 +104,11 @@ void doDepositionShapeN(const GetParticlePosition& GetPosition,
     amrex::ParallelFor(
         np_to_depose,
         [=] AMREX_GPU_DEVICE (long ip) {
+#if (defined AMREX_USE_GPU)
+            KernelTimer KnlTimer(cost && load_balance_costs_update_algo
+                                 == LoadBalanceCostsUpdateAlgo::GpuClock, cost);
```

Comment:
@mrowan137 this KernelTimer initialization should also depend on whether it's a load balance interval